app/src/main/java/space/taran/arknavigator/mvp/presenter/adapter/ResourcesGridPresenter.kt

```diff
@@ -79,6 +84,7 @@ class ResourcesGridPresenter(
 
     suspend fun updateSelection(selection: Set<ResourceId>) = withContext(Dispatchers.Default) {
         this@ResourcesGridPresenter.selection = resources.filter { selection.contains(it.id) }
```

Comment:
maybe it's a better idea to have this filtration of resources to be taken care from another class which will serve as a filter for this kind of things. Basically the job of the presenter is to do the logic(not to be done on the view). Also if we extract these kind of logics to separate classes we will have cleaner code, separation of concerns, and unit tests will be more easy to do.

Reply:
@darkovski88 actually we have such an entity, _tags selector_. Classes `TagsSelectorPresenter` and `TagsSelectorAdapter`. Maybe the architecture can be improved here.

Reply:
@kirillt as i can see here the code and app looks ok to me. I have no negative comments on this PR except maybe take some action to the ones that i've already added